tensorflow_asr/losses/rnnt_losses.py

```diff
@@ -14,18 +14,20 @@
 # RNNT loss implementation in pure TensorFlow is borrowed from [iamjanvijay's repo](https://github.com/iamjanvijay/rnnt)
 
 import tensorflow as tf
+from tensorflow.python.ops.gen_array_ops import matrix_diag_part_v2
+
 try:
     from warprnnt_tensorflow import rnnt_loss as warp_rnnt_loss
     use_warprnnt = True
 except ImportError:
     print("Cannot import RNNT loss in warprnnt. Falls back to RNNT in TensorFlow")
     print("Note: The RNNT in Tensorflow is not supported for CPU yet")
-    from tensorflow.python.ops.gen_array_ops import matrix_diag_part_v2
     use_warprnnt = False
 
 
 def rnnt_loss(logits, labels, label_length, logit_length, blank=0, name=None):
     if use_warprnnt:
+        from tensorflow.python.ops.gen_array_ops import matrix_diag_part_v2
```

Comment:
@gandroz I don't think the import is needed here.

Reply:
Your are right, it is a duplicate, my bad. We have to choose wether to import the module here or to import it at the beginning of the module, even if it is not used. It is cleaner at the beginning but it could be useless (unless the cuda version is discarded).

It is needed for the test function as I explicitely set the `use_warprnnt` flag [here](https://github.com/TensorSpeech/TensorFlowASR/blob/ed1b117220231d004ff61f75041cf7823e5c03e2/tests/losses/test_rnnt_loss.py#L75).

Reply:
I have some update in this file, please resolve the conflict :smile: